genie-agent/src/main/resources/application.yml

```diff
@@ -5,3 +5,7 @@ cloud:
       instanceProfile: true
     stack:
       auto: false
+
+spring:
```

Comment:
This might produce the same end result in terms of stuff loaded, but is it also the same in terms of speed?
Just curious and wondering if you did this benchmark.